micrometer-spring-legacy/src/main/java/io/micrometer/spring/autoconfigure/export/azure/AzureProperties.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2017 Pivotal Software, Inc.
```

Comment:
perhaps the year in copyright can be changed to the 2018?

Reply:
@jkschneider perhaps its the appropriate suggestion from Sergey. We should update copyright to 2018. Let me know.